contracts/Front/Uploads.sol

```diff
@@ -43,8 +45,14 @@ contract Uploads is Base {
         sheltering.storeBundle(bundleId, msg.sender, storagePeriods);
         (uint challengeFee, uint validatorsFee) = fees.calculateFeeSplit(msg.value);
         block.coinbase.transfer(validatorsFee);
-        challenges.startForSystem.value(challengeFee)(msg.sender, bundleId, 7);
+        challenges.startForSystem.value(challengeFee)(msg.sender, bundleId, SYSTEM_CHALLENGES_COUNT);
 
         emit BundleUploaded(bundleId, storagePeriods);
     }
+
+    function triggerSystemChallenges(bytes32 bundleId) public payable {
```

Comment:
This mechanism won't work. You need to implement proper removing shelterers first.

Reply:
Basically it depends if we want to implement mechanism allowing only to retrigger initial challenges, or retrigger them not only at the beginning, but also after all shelterings are expired and Hermes want his bundle back to life.